.github/workflows/create-release.yml

```diff
@@ -39,3 +39,9 @@ jobs:
       run: node ./bin/create-github-release.js --tag ${{ steps.get_tag.outputs.latest_tag }} --repo-owner ${{ github.repository_owner }}
       env:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    - run: npm run public-docs
+    - name: Deploy Docs
+      uses: peaceiris/actions-gh-pages@v3
+      with:
+        github_token: ${{ secrets.GITHUB_TOKEN }}
+        publish_dir: ./out
```

Comment:
Curious why we aren't using `npm run publish-docs`?

Could be it doesn't work, just want to understand.

My only concern being using CI specific mechanisms can make things harder in the long run when you occasionally have to do things manually, need to debug, etc. Ideal state to me is to keep local/CI runs as close together as possible and leverage specific CI platform just for the specific glue needed. Not always possible, but a guidepost.

Reply:
Valid, I'll make the change.

Reply:
My only concern is: do we know that the CI box will have appropriate credentials? Do we need to make any changes so that it does?

Reply:
Probably something that could be tested out on a fork like I did with the prepare release script.

That being said, the prepare release workflow is a good example of how to setup the credentials needed for pushing code and whatnot: https://github.com/newrelic/node-newrelic/blob/main/.github/workflows/prepare-release.yml#L28-L35

It sets up the GitHub user for doing commits and ensures the token is set in the ENV for the API calls. I'm guessing this only needs the creds setup but don't know for sure w/o testing.